mcs/class/Mono.Posix/Mono.Unix.Native/Stdlib.cs

```diff
@@ -378,7 +380,11 @@ static XPrintfFunctions ()
 	//
 	public class Stdlib
 	{
+#if NETSTANDARD2_0
+		internal const string LIBC = "c";
```

Comment:
@bholmes this still doesn't solve the .NET Core on Windows use case right? Or do they remap `[DllImport("c")]` 
 to msvcrt ?